generic/utils/ras.py.data/common_func.sh

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
+################################################################################
+# Common functions to all RAS testcase scripts
+#
+# Author: Lucas Meneghel Rodrigues (lucasmr@br.ibm.com)
```

Comment:
@Pavithra1602 do you know the actual license of this file? @lmr was it GPL compatible, if not could you please also release it under GLPv2+?